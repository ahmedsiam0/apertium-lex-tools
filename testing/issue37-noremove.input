^,<cm>/,<cm>$ ^inlem<a>/selectfallback<b>/notselected<c>/removeme<d>/notselectedeither<e>$
