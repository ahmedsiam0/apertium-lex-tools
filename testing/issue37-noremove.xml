<rules>

  <rule weight="1.61803">
    <match lemma="inlem" tags="a"><select lemma="selectfallback"/></match>
  </rule>

  <rule weight="3.14159">
    <match tags="*"/>
    <match lemma="inlem" tags="a"><remove lemma="removeme"/></match>
  </rule>

</rules>
