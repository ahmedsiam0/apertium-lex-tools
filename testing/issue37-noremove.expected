^,<cm>/,<cm>$ ^inlem<a>/selectfallback<b>$
